### DISCUSSION TITLE
Ok, Im pretty new to this but Dont be to Harsh

### DISCUSSION THREAD

**Discussion by Melynxis** [trickerer/Trinity-Bots#852] [General]
Ok, I am totally new to the whole TrinityCore/Trinity Bots project, and have gotten the system to work successfully.  I followed to the T a Guide i found on a website (Can be provided if needed) and everything at least core game seems to be working perfectly.  The problem I have ran into is I dont seem to have any Bots at all, I can spawn the hiring Bot, but it shows no bots available for hire in the region.  Should these bots be prepopulated somewhere? Did i do something wrong? Should the hiring Person be empty? Is this something i can fix? Lol So confused.. any help is appreciated though.

**Comment by Melynxis** [comment]
https://www.ustoopia.nl/games/build-a-private-wow-wotlk-server-with-npc-bots-on-ubuntu-22-04/ specifically the link followed

**Comment by trickerer** [comment]
Bots have to be spawned in order to make them available for hire either using botgiver npc or directly by talking to them. Two main command you'll need to do that are `.npcbot lookup` and `.npcbot spawn`. I understand Readme for bots is TLDR but it explains pretty much everything and describes both spawn process and the commands in detail.

**Reply by Melynxis** [reply]
Awesome, that actually helps alot, Im using the Netherbot plugin, and had been looking through stuff and didnt know if I'd somehow messed something up or not :) Basically I just wanted to confirm if i needed to go in and Prespawn everything
